tests/test_transformations.py

```diff
@@ -86,33 +86,33 @@ def test_video_downsampling(input_fps, output_fps, random_seed, strategy,
     assert np.array_equal(expected_video, new_video)
 
 
-@pytest.mark.parametrize(("coordinate_pair, window_size, video_shape,"
-                          "expected_pair"), [
-                        ((0, 9), (10, 10), (512, 512), (4, 9)),
-                        ((9, 0), (10, 10), (512, 512), (9, 4)),
-                        ((511, 4), (10, 10), (512, 512), (506, 4)),
-                        ((4, 511), (10, 10), (512, 512), (4, 506)),
-                        ((5, 5), (3, 3), (5, 5), (3, 3)),
-                        ((-1, 2), (3, 3), (5, 5), (1, 2))])
-def test_coordinate_window_shift(coordinate_pair, window_size, video_shape,
-                                 expected_pair):
-    transformed_center = transformations.get_transformed_center(coordinate_pair,
-                                                                     window_size,
-                                                                     video_shape)
-    assert transformed_center == expected_pair
-
-
-@pytest.mark.parametrize(("coordinate_pair, window_size, video_shape,"
-                          "expected_pair"),[
-                        ((3, 3), (600, 600), (512, 512), None)
-                        ])
-def test_coordinate_shift_exception(coordinate_pair, window_size, video_shape,
-                                    expected_pair):
-    with pytest.raises(ValueError):
-        transformed_center = transformations.get_transformed_center(coordinate_pair,
-                                                                    window_size,
-                                                                    video_shape)
-        assert transformed_center == expected_pair
+# @pytest.mark.parametrize(("coordinate_pair, window_size, video_shape,"
+#                           "expected_pair"), [
+#                         ((0, 9), (10, 10), (512, 512), (4, 9)),
+#                         ((9, 0), (10, 10), (512, 512), (9, 4)),
+#                         ((511, 4), (10, 10), (512, 512), (506, 4)),
+#                         ((4, 511), (10, 10), (512, 512), (4, 506)),
+#                         ((5, 5), (3, 3), (5, 5), (3, 3)),
+#                         ((-1, 2), (3, 3), (5, 5), (1, 2))])
```

Comment:
I would consider adding a test case such that coordinate_pair (+,-) for complete coverage.